browser/src/control/AutoCompletePopup.ts

```diff
@@ -162,8 +162,21 @@ abstract class AutoCompletePopup {
 		data.children[0].children[0] = control;
 		(data.children[0].children[0] as TreeWidget).entries = entries;
 
-		data.posx = cursorPos.x;
-		data.posy = cursorPos.y;
+		const isSpreadsheetRTL = this.map._docLayer.isCalcRTL();
```

Comment:
can't this be handled by the Control.JSDialog code?

I think it should work the same as other cases, we adjust RTL and limit to screen edge there.